### DISCUSSION TITLE
Can not buildproject, not find (import libv2ray.Libv2ray and  import go.Seq) in SagerNet.kt )

### DISCUSSION THREAD

**Discussion by octasync** [SagerNet/SagerNet#119] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/63095166/120176761-195a7500-c221-11eb-84c4-4aa0a687d01c.png)
Please help me!

**Comment by nekohasekai** [comment, marked as answer]
Refer to workflow to compile

**Reply by octasync** [reply]
Sorry, i dont understand, can you tell me instruction?

**Reply by octasync** [reply]
How refer to workflow to compile?

**Reply by nekohasekai** [reply]
https://github.com/SagerNet/SagerNet/tree/dev/.github/workflows

**Reply by octasync** [reply]
Sorry again for bothering you, but how to compile using .yml workflows?

**Reply by octasync** [reply]
I am ashamed

**Reply by nekohasekai** [reply]
Understand the github action compilation process on your own. Are you not an android developer?

**Reply by muzaffer72** [reply]
workflows Not working. I have build error

**Reply by muzaffer72** [reply]
![resim](https://user-images.githubusercontent.com/10040551/182483280-27ec5150-0f57-418a-82b3-a1c08f161a6c.png)

**Reply by lastmansurviving13** [reply]
hi comrads
did anyone found the solution?
what does refer to workflow exactly mean?
